src/rabbitizer/Cargo.toml

```
@@ -57,6 +57,7 @@ doc = ["MIPS_IV", "all_extensions", "all_gated_extensions"]
 
 [dependencies]
 bitflags = { version = "2.6.0", default-features = false }
+ordered-float = { version = "5.0.0", default-features = false }
```

Since this is only used by allegrex then could you make it optional and only enable it when that feature is on?
```suggestion
ordered-float = { version = "5.0.0", default-features = false, optional = true }
```

and add `"dep:ordered-float"` to `R4000ALLEGREX`